libmattermost.h

```diff
@@ -35,6 +35,8 @@
 #define MATTERMOST_USER_PAGE_SIZE 200 // 200 is MAX. in paged queries (and default)
 #define MATTERMOST_HISTORY_PAGE_SIZE 60 // 200 is MAX in paged queries (60 is default)
 #define MATTERMOST_MAX_PAGES 10 // that is 2000 users or posts in paged queries
+#define MATTERMOST_NEW_CHANNEL_FOUND -2 // -2 indicates that timestamp does not exist for a new channel
```

Comment:
Move this line to where other timestamp values are declared.

Reply:
there is no any existing constant for timestamp, however moved it now onto a separate line